### PR TITLE
Release: Update nydus snapshotter version

### PR DESCRIPTION
**Pull request by portersrc**
This is updated to match the nydus snapshotter version (v0.13.11) from the v3.4.0 kata release.

This is needed for v0.9.0 release process.

### CODE REVIEW

File: install/pre-install-payload/Dockerfile [hunk 1]
```diff
@@ -69,10 +69,9 @@ ENV GOARCH=${ARCH}
...
 RUN mkdir -p ${NODE_DESTINATION}/bin && \
     apk add --no-cache curl && \
-    if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
-    curl -fOL --progress-bar ${NYDUS_SNAPSHOTTER_REPO}/releases/download/${NYDUS_SNAPSHOTTER_VERSION}/nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz && \
-    tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz -C / && \
-    rm nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz && \
+    curl -fOL --progress-bar ${NYDUS_SNAPSHOTTER_REPO}/releases/download/${NYDUS_SNAPSHOTTER_VERSION}/nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz && \
+    tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz -C / && \
+    rm nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz && \
     mv /nydus-snapshotter/* ${NODE_DESTINATION}/bin/ && \
```
Comment (ldoktor) [inline install/pre-install-payload/Dockerfile:75]: The v0.13.11 doesn't use the `nydus-snapshotter` prefix inside the tarball so you need to remove the lines `75,76` (mv and rm /nydus-snapshotter)

**Review by ldoktor — COMMENTED** [review]
thanks @portersrc, this changes the repo to the official one, which is in sync with kata-containers. The only little issue is that that version doesn't use the prefix inside the tarball but is directly uncompressed as it should. Once you remove the 2 extra lines it should be good to go.

File: install/pre-install-payload/Dockerfile (hunk 1 above)
Comment (portersrc) [reply install/pre-install-payload/Dockerfile]: This is a good catch. I think I made a slightly different but appropriate change and see enclave-cc test passing now

**Review by fitzthum — APPROVED** [review]
LGTM

File: install/pre-install-payload/Dockerfile [hunk 2]
```diff
@@ -69,11 +69,11 @@ ENV GOARCH=${ARCH}
...
-    if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
-    curl -fOL --progress-bar ${NYDUS_SNAPSHOTTER_REPO}/releases/download/${NYDUS_SNAPSHOTTER_VERSION}/nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz && \
-    tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz -C / && \
-    rm nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz && \
-    mv /nydus-snapshotter/* ${NODE_DESTINATION}/bin/ && \
+    curl -fOL --progress-bar ${NYDUS_SNAPSHOTTER_REPO}/releases/download/${NYDUS_SNAPSHOTTER_VERSION}/nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz && \
+    mkdir /nydus-snapshotter && \
+    tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz -C /nydus-snapshotter && \
+    rm nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-linux-${ARCH}.tar.gz && \
+    mv /nydus-snapshotter/bin/* ${NODE_DESTINATION}/bin/ && \
```
[the remark this answers is not in the document]
Comment (ldoktor) [reply install/pre-install-payload/Dockerfile:76]: Note it's not a strict requirement, just my preference. So feel free to say you really really want it this way and I will get along with this complicated construct...

[the remark this answers is not in the document]
Comment (ldoktor) [reply install/pre-install-payload/Dockerfile:76]: Btw it must be extracted to `tar xvzpf nydus-snapshotter-${NYDUS_SNAPSHOTTER_VERSION}-${ARCH}.tgz -C ${NODE_DESTINATION}/bin/ && \` then everything works well...

**Review by wainersm — APPROVED** [review]
LGTM. Thanks @portersrc !

File: install/pre-install-payload/Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (portersrc) [reply install/pre-install-payload/Dockerfile]: 👍  thanks @ldoktor. I incorporated this suggested now, and it seems to still be passing (just awaiting one last runner again).

**Comment by wainersm** [comment]
Hi @portersrc ! Can I get this merged or are you waiting on some other PR?

**Review by ldoktor — APPROVED** [review]
Thanks, simple is better :-)

**Comment by portersrc** [comment]
Thanks, guys. Yes this was OK to merge. We can perform the rest of v0.9.0 steps now, I believe.

**Pull request merged**